buildtest/cli/__init__.py

```diff
@@ -953,6 +983,13 @@ def report_menu(subparsers):
         action="store_true",
         help="Print output in machine readable format",
     )
+    parser_report.add_argument(
+        "--color",
+        type=supported_color,
+        metavar="COLOR",
+        action="append",
+        help="Prints reports and the subcommand summary in the provided COLOR, colors defined in Rich.Color",
```

Comment:
You can say `help="Print output of report table with the selected color.`

That way we are not too specific on the help message since implementation may change in future. We can elaborate on this in the user docs instead of the help message.

Reply:
simplified